decidim-core/lib/decidim/participable.rb

```diff
@@ -82,7 +82,7 @@ def participatory_space_manifest
       #
       # Returns an `ActiveRecord::Association`.
       def public_spaces
-        published
+        published.where(private_space: false)
```

Comment:
Will this keep working if the space has no `private_space` field? Initiatives, for example.